### DISCUSSION TITLE
Preload for css is found, but not used, because request credentials mode does not match

### DISCUSSION THREAD

**Discussion by howells** [vercel/next.js#16850] [Help]
I've only just started getting this problem with a new nextjs app, where css files aren't preloaded, and therefore not used at all:

```
A preload for 'https://*******.vercel.app/_next/static/css/db098983a22a0aa338df.css' is found, but is not used because the request credentials mode does not match. Consider taking a look at crossorigin attribute.
```

I'm using sass modules, and this is with nextjs 9.5.3 and webpack 5 (but the same issue occurs with webpack 4). Any ideas?

**Comment by howells** [comment]
Just discovered this doesn't happen when I downgrade to 9.5.2, so it's a 9.5.3 issue.

**Reply by andrsdev** [reply]
I confirm this!

**Reply by howells** [reply]
Glad it's not just me - are you able to create an issue? I don't feel confident enough I know what the issues is to produce a reproducible repo

**Reply by m-sheeraz-it** [reply]
Yes. I also have the same error with next 9.5.3

**Reply by subwaymatch** [reply]
Same error in 9.5.3

**Reply by maickoldxd** [reply]
I'm still having this error in 9.5.2, also I have bootstrap + next sass

**Reply by egemon** [reply]
Error with 9.5.2

**Reply by vinamelody** [reply]
Also having this in 9.5.5

**Reply by niko20** [reply]
Same error in 10.0.0 with my app compiled

**Comment by timtyrrell** [comment]
😎 

<img width="787" alt="Screen Shot 2020-10-27 at 9 47 46 AM" src="https://user-images.githubusercontent.com/100535/97326596-c3ac8d00-1839-11eb-9159-27756de9c5e8.png">

**Comment by cesarkohl** [comment]
I've added the attribute/value `crossOrigin="anonymous"` and the warning has gone.

`<link rel="preload" href="/fonts/font.woff2" as="font" crossOrigin="anonymous" />`

**Reply by alitnp** [reply]
lifesaver.  thanks ```crossOrigin="anonymous"``` does the job.

**Reply by deepakchandel12** [reply]
I think it will work but just describe step. There are come many people and they not know How to add the attribute or value for fix this error. Please tell us fully or provide videos for easy. My website freefontsstore.com and there are not showing adsense ads because showing error preloader error, j error in insight..

**Comment by maickoldxd** [comment]
to have a crossOrigin="anonymous" in all link tags,
edit your next.config.js:
`
//If you have others plugins, use next-compose-plugins
const withPlugins = require('next-compose-plugins'); 
//In my case I use sass & pwa
const withSass = require('@zeit/next-sass')
const withPWA = require('next-pwa')

module.exports = withPlugins([withSass(
  withPWA({
      pwa: {
          dest: 'public'
      }
  })
)],{
//Here you define your next js and webpack config
crossOrigin: 'anonymous',
}
`

**Comment by hirasso** [comment]
Only adding the attribute ~crossover~ `crossorigin` without any value should be enough already, as described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preload#cors-enabled_fetches

**Reply by johnjensenish** [reply]
I assume you mean the attribute `crossorigin`, and mentioning it for posterity.

I believe React/JSX requires `crossOrigin="anonymous"` because it doesn't treat that prop as a boolean.

**Reply by hirasso** [reply]
Oh right, of course `crossorigin` :) Thanks for pointing it out.

**Comment by guotie** [comment]
strange problem, why?
